### PR TITLE
Add fragment add icon code field 5.0.1

### PR DESCRIPTION
**Pull request by fedelocci**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: package-lock.json [hunk 1]
```diff
@@ -19,56 +19,56 @@
       "integrity": "sha512-Il19yJvy7vMFm8AVAh6OZzaFoAd0hbkeMZiX3P5HGD+z7dyI7RzndHB0dg6Urh/VAFfHtpOIzDUSxmY6coyZWQ==",
       "dev": true,
       "requires": {
-        "chalk": "^2.0.0",
-        "esutils": "^2.0.2",
-        "js-tokens": "^3.0.0"
+        "chalk": "2.3.2",
```
Comment (alessioscalici) [inline package-lock.json]: There is no need for this package-lock.json change. Can you please remove?

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**